examples/zsl.cpp

```diff
@@ -22,20 +22,20 @@ int main(int argc, char ** argv) {
     }
 
     // load the image
-    std::string img_path = params.image_paths[0];
+    const char *img_path = params.image_paths[0].c_str();
```

Comment:
const auto&. c-api and c-like-code are different things.

Reply:
Fixed.